### DISCUSSION TITLE
project run slow

### DISCUSSION THREAD

**Discussion by iron4rain** [dropways/deskapp#52] [General]
### Steps to reproduce

1.  click menu selection

### Expected result

1.  switch menu page to another page  quickly

### Actual result

1.  almost cost 1~2 sec on loading 

CPU 9750H GPU 1660ti-maxq
your project very beautifal, but a little slow to use.

**Comment by Ahmadnew** [comment]
Hey Bro Just Remove Core.min and style.min In Vendors Folder And Check Speed
Note : There Will Be No Effect In Website Styling
